tests/fixtures/1_single_zone_tiny_house.json

```diff
@@ -0,0 +1,961 @@
+{
+    "type": "Model",
+    "name": "TinyHouse",
+    "name_original": "Tiny House",
+    "properties": {
+        "type": "ModelProperties",
+        "energy": {
+            "type": "ModelEnergyProperties",
+            "global_construction_set": {
```

Comment:
Could be possible to make this a reference into the construction_sets array

Reply:
Now that you mention it, I think you are right that a reference is the right way to do it.  I think I'll go ahead and implement it once @mostaphaRoudsari gets the chance to do an initial review.

Reply:
It can go either way. I think the current design protects the global / default construction from being overwritten or mixed with user / model construction sets.

Reply:
I implemented this as a named referenced in the updates samples.